src/components/sound.js

```diff
@@ -173,7 +173,7 @@ module.exports.Component = registerComponent('sound', {
     for (i = 0; i < this.pool.children.length; i++) {
       sound = this.pool.children[i];
       sound.onEnded = function () {
-        this.isPlaying = false;
+        self.isPlaying = false;
```

Comment:
Question for whomever does review this from the AFrame team.

Are we explicitly avoiding arrow functions to capture this?

If we are, should we instead be using the bind() helper instead of capturing a "self"? This would make things more consistent. We tend to use bind() for simple functions and any complex function wrappers we instead use self.

Reply:
Yes, no arrow functions: Not supported in IE 11. As @DigiTec said probably bind a bit cleaner than self.